dotnet-client-libraries/HandlingDatesWithNodaTime/HandlingDatesWithNodaTime/Program.cs

```diff
@@ -0,0 +1 @@
+﻿Console.WriteLine("Application started");
```

Comment:
We should either show something in the console application, or we should remove it.

Reply:
I've opted to remove it entirely. Please review

Reply:
Actually, the pipeline has failed after removing the statement. Can I show the current time in the console in stead?

Reply:
It wouldn't have built locally for you either. We need to be sure we build locally before we push a commit.
In regards to the question, I had not looked at the article at all yet, so was just proposing that we either do something in the console app OR remove the console app project all together. Now that I have taken a quick look at the article and since we are showing how to create a console app, then we need to keep it. The code samples in the article seem to be something that we would be running in the console app, so we should be demoing each of those things within the console app.